lib/signature.coffee

```diff
@@ -93,10 +95,13 @@ module.exports = class Signature
 				if parameter.isOptional() and _.isEmpty(value)
 					return result
 
-				result[parameter.getValue()] = value
+				result[parameterValue] = value
 				return result
 
 			if not parameter.isWord() and word?
-				result[parameter.getValue()] = _.parseInt(word) or word
+				if /^\d+$/.test(word)
```

Comment:
Yeah, I guess numbers with spaces in is unlikely, they'd have to `" 1234 "` or `"1 234"` for that to happen, both of which are unlikely